ui/StatusQ/tests/tst_ConcatModel.cpp

```diff
@@ -17,8 +17,9 @@
 #include <StatusQ/concatmodel.h>
 
 namespace {
-
-// Workaround for a bug in QIdentityProxyModel (returning roleNames improperly)
+// Workaround for https://bugreports.qt.io/browse/QTBUG-57971 (ListModel doesn't
+// emit modelReset when role names are initially set, therefore QIdentityProxyModel
+// doesn't update role names appropriately)
 class IdentityModel : public QIdentityProxyModel {
```

Comment:
Still need this as a `QIdentityProxyModel` now that you changed the impl to `QAIM`?

Reply:
Yes, it's needed here, only for test. Bc in tests I'm using ListModel and the bug needs this special handling.